src/commands/config.ts

```diff
@@ -0,0 +1,15 @@
+import { getConfiguration, getGlobalGithubConfig } from '../config';
+import { formatJson } from '../utils/strings';
+
+export const command = ['config'];
+export const description =
+  'Print the parsed, processed, and validated Craft config for the current project in pretty-JSON.';
+
+export async function handler(): Promise<void> {
+  const github = await getGlobalGithubConfig();
+  const config = {
+    ...getConfiguration(),
+    github,
```

Comment:
>As written, this takes the inferred info over an explicit config. Is that what we want? Can we abstract out this logic if it's used in multiple places so that we are consistent?

The "inferred" one is still looking at the config first and then tries to infer. This is already abstracted away in #218. I'd love to bake this into `getConfiguration()` itself but one is sync and one is async so it may require a larger refactor. 	I'll look into this in a follow-up patch.

Reply:
Confirmed that config overrides inferred:

```
$ ./dist/craft config | jq .github.repo
"cruft"
$ gd
```
```diff
diff --git a/.craft.yml b/.craft.yml
index b2be798..01cad7e 100644
--- a/.craft.yml
+++ b/.craft.yml
@@ -1,7 +1,7 @@
 minVersion: '0.21.0'
 github:
   owner: getsentry
-  repo: craft
+  repo: cruft
 changelogPolicy: auto
 requireNames:
   - /^sentry-craft.*\.tgz$/
```